parquet-cli/src/main/java/org/apache/parquet/cli/BaseCommand.java

```diff
@@ -305,11 +305,11 @@ protected static ClassLoader loaderForPaths(List<String> paths)
         // TODO: add these to the reader builder
         AvroReadSupport.setRequestedProjection(conf, projection);
         AvroReadSupport.setAvroReadSchema(conf, projection);
-        final ParquetReader<D> parquet = AvroParquetReader.<D>builder(qualifiedPath(source))
+        try(final ParquetReader<D> parquet = AvroParquetReader.<D>builder(qualifiedPath(source))
             .disableCompatibility()
             .withDataModel(GenericData.get())
             .withConf(conf)
-            .build();
+            .build()) {
         return new Iterable<D>() {
```

Comment:
Do you not indent here?

Reply:
My IDE didn't, do you think we should indent here?